packages/contracts/docs/osx/02-how-to-guides/01-dao/02-action-execution.md

```diff
@@ -1,15 +1,77 @@
 ---
-title: Executing Actions
+title: How to execute an action on behalf of the DAO
 ---
 
 ## Using the DAO Executor
 
-For the `execute` call to work, the `msg.sender` must have the [`EXECUTE_PERMISSION_ID` permission](../../01-how-it-works/01-core/02-permissions/index.md#permissions-native-to-the-dao-contract) on the DAO contract.
+Executing actions on behalf of the DAO is done through the `execute` function from the `DAO.sol` contract. This function allows us to [pass any encoded list (or array of actions)](https://github.com/aragon/osx/blob/develop/packages/contracts/src/core/dao/DAO.sol#L168) to be executed by the DAO contract itself.
```

Comment:
Why "encoded list"?

Moreover, the link you inserted there is not a permanent link. It is the link to `DAO.sol#L168` that changes with every change we apply to DAO.